extra/html-extra/HtmlExtension.php

```diff
@@ -79,6 +87,35 @@ public function dataUri(string $data, string $mime = null, array $parameters = [
 
         return $repr;
     }
+
+    /**
+     * @param array{string, string|bool|int|float|null} $attributes
```

Comment:
wrong type. `array{string, string|bool|int|float|null}` means an array with 2 elements, with index 0 being a string and index 1 being `string|bool|int|float|null`. Instead, you should use `array<string, string|bool|int|float|null>`